src/gt4py/backend/gtc_backend/defir_to_gtir.py

```diff
@@ -259,8 +266,12 @@ def visit_AxisBound(self, node: AxisBound):
         )
 
     def visit_FieldDecl(self, node: FieldDecl):
+        domain_axes = Domain.LatLonGrid().axes_names
```

Comment:
Please just use the names here and remove the dependency from the old pipeline. The phrase axes should disappear altogether.

Reply:
I will do this, even though I think it's a bit excessive, seeing as the `DefIR` is part of the old toolchain. A visitor lowering from it will always have dependencies on the old toolchain, therefore. I assume reducing the usage of `axes` is worth it.

Reply:
(note also that the names in `dimension_flags_to_names` are not and do not need to be the same as the ones here).